topics/single-cell/tutorials/scatac-standard-processing-snapatac2/tutorial.md

```diff
@@ -0,0 +1,940 @@
+
+Luckily, the marker genes for {PBMC's} are known and can be found in databases such as [PanglaoDB](https://panglaodb.se/markers.html) ({% cite Franzn2019 %}). Thus, we can annotate our clusters manually using known marker genes. 
+
+## Gene matrix
+Since our data currently doesn't contain gene information, we have to create a cell by gene activity matrix using the function *pp.make_gene_matrix*. 
+
+> <hands-on-title> Task description </hands-on-title>
+>
+> 1. {% tool [SnapATAC2 Preprocessing](toolshed.g2.bx.psu.edu/repos/iuc/snapatac2_preprocessing/snapatac2_preprocessing/2.5.3+galaxy1) %} with the following parameters:
+>    - *"Method used for preprocessing"*: `Generate cell by gene activity matrix, using 'pp.make_gene_matrix'`
+>        - {% icon param-file %} *"Annotated data matrix"*: `Anndata 5k PBMC leiden` (output of **tl.leiden** {% icon tool %})
+>        - {% icon param-file %} *"GTF/GFF file containing the gene annotation"*: `gene_annotation.gtf.gz` (Input dataset)
+> 2. Rename the generated file to `Anndata 5k PBMC gene_matrix` or add the tag {% icon galaxy-tags %} `gene_matrix` to the dataset
+>    > <tip-title> Gene matrix </tip-title>
+>    >
+>    > - Please note that *pp.make_gene_matrix* removes all annotations except those stored in `obs`. 
+>    > - Therefore it might be necessary to remove propagating tags {% icon galaxy-tags %} (tags starting with `#`) from `Anndata 5k PBMC gene_matrix`. 
+>    >    - Tags can be removed by expanding the dataset with a tag and clicking the `x` next to the tag.
+>    {: .tip}
+>
+> 3. {% icon galaxy-eye %} Inspect the general information of the `.h5ad` output
+>
+>    > <question-title></question-title>
+>    >
+>    > ```
+>    > AnnData object with n_obs × n_vars = 4430 × 60606
+>    >  obs: 'n_fragment', 'frac_dup', 'frac_mito', 'tsse', 'doublet_probability', 'doublet_score', 'leiden'
+>    > ```
+>    >
+>    > What does `n_vars` represent in `Anndata 5k PBMC gene_matrix` and what did it represent in `Anndata 5k PBMC leiden`?
+>    >
+>    > > <solution-title></solution-title>
+>    > >
+>    > > - The variables now represent accessible genes. There are 60606 accessible genes in our samples. In `Anndata 5k PBMC leiden` and all earlier AnnData the variables represented the 6062095 fixed-sized genomic bins. 
+>    > >
+>    > {: .solution}
+>    >
+>    {: .question}
+>
+{: .hands_on}
+
+
+## Imputation with scanpy
+Similar to scRNA-seq data, the cell by gene activity matrix is very sparse. Additionally, high gene variance between cells, due to technical confounders, could impact the downstream analysis. In scRNA-seq, filtering and normalization are therefore required to produce a high-quality gene matrix. 
+
+Since the cell by gene activity matrix resembles the cell by gene expression matrix of scRNA-seq, we can use the tools of the [Scanpy](https://scanpy.readthedocs.io/en/stable/index.html) ({%cite Wolf2018%}) tool suite to continue with our data. 
```

Comment:
Please re-check this sentense. There is "cell by gene" two times.

Reply:
The difference is scATAC-seq produces cell-by-gene-activity matrices and scRNA-seq produces cell-by-gene-expression matrices

Reply:
Sorry, I overlooked it.